Dockerfile.dev

```diff
@@ -67,14 +67,15 @@ ENV AIRFLOW_HOME=/app \
     AIRFLOW_DATABASE_URL=mysql://root:secret@db:3306/airflow \
     AIRFLOW__CORE__FERNET_KEY="0000000000000000000000000000000000000000000=" \
     AIRFLOW_SECRET_KEY="000000000000000000000000000000000000000000=" \
-    # AIRFLOW_SMTP_HOST= \
-    # AIRFLOW_SMTP_USER= \
-    # AIRFLOW_SMTP_PASSWORD= \
     AIRFLOW_SMTP_FROM=telemetry-alerts@airflow.dev.mozaws.net \
     AIRFLOW__SCHEDULER__CATCHUP_BY_DEFAULT=False \
     AIRFLOW_EMAIL_BACKEND="airflow.macros.log_email_backend.log_email_backend" \
+    AIRFLOW__KUBERNETES__IN_CLUSTER=False\
```

Comment:
in_cluster previously defaulted to False in 1.10.2, but this changed with 1.10.7 (bugs), and I added a default in our GKEPodOperator as well as this env config because some code paths check env and others use the argument iirc